todo_app/app.py

```diff
@@ -40,5 +41,4 @@ def isvalid(new_todo_item):
     return app
 
 if __name__ == '__main__':
-    app = create_app()
-    app.run()
+    app = create_app().run(host='0.0.0.0', debug=True)
```

Comment:
You shouldn't need the `host='0.0.0.0'` - you already specify it in your dockerfile entrypoints